### PR TITLE
Database and realtime user setup for signup page

### PR DESCRIPTION
**Pull request by neha702**

### CODE REVIEW

File: app/src/main/java/com/Executioner/meetx/MainActivity.java [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 
 import com.Executioner.meetx.authentication.Login_page;
 import com.Executioner.meetx.authentication.Signup_page;
 
 public class MainActivity extends AppCompatActivity {
+    String TAG="pratiksha";
```
Comment (Pratikkumar24) [inline app/src/main/java/com/Executioner/meetx/MainActivity.java:16]: Always try to make meaningful TAG's, Like String TAG = "SignUp_page"

Comment (neha702) [reply app/src/main/java/com/Executioner/meetx/MainActivity.java:16]: You also wrote pratik na !

Comment (Pratikkumar24) [reply app/src/main/java/com/Executioner/meetx/MainActivity.java:16]: Okay na!! that is for everyone

**Review by Pratikkumar24 — APPROVED** [review]
Please make good tags from next time

**Pull request merged**